rc/lsp.kak

```diff
@@ -1,6 +1,9 @@
-decl str lsp_cmd 'nc localhost 31337'
+decl str lsp_cmd 'nc localhost 31337 -c'
 decl -hidden completions lsp_completions
-decl -hidden range-specs lsp_errors 
+decl -hidden range-specs lsp_errors
+
+#cquery
```

Comment:
nit:  option name already has self-explanatory prefix `cquery` so we can skip comment here